storage/__init__.py

```diff
@@ -1,8 +1,7 @@
-from .storage import get_storage  # noqa: F401
-from .storage import register_storage_protocol  # noqa: F401
-from .swift_storage import register_swift_protocol  # noqa: F401
-from . import local_storage  # noqa: F401
-from . import swift_storage  # noqa: F401
-from . import ftp_storage  # noqa: F401
-from . import s3_storage  # noqa: F401
-from . import google_storage  # noqa: F401
+from storage.storage import get_storage, register_storage_protocol  # noqa: F401
+# from .swift_storage import register_swift_protocol  # noqa: F401
```

Comment:
Actually this may be a good catch in that we *do* need it -- if we don't expose register_swift_protocol in this scope, that's a backwards-incompatible API change. Obviously disabling Python 2 is a backwards-incompatible change in and of itself, but this would be library-level. (Tough to know if anyone else uses it, although considering the many Swift implementations it's not unreasonable... :)